plugins/qsi-dock/qsi-dock.quirk

```diff
@@ -0,0 +1,9 @@
+[USB\VID_2BEF&PID_1003]
```

Comment:
is 2BEF super new? I don't see anything in https://devicehunt.com/view/type/usb/vendor/2BEF